homeassistant/components/icloud/device_tracker.py

```diff
@@ -27,7 +27,7 @@ async def async_setup_scanner(
     config: ConfigType,
     see: Callable[..., Awaitable[None]],
     discovery_info: DiscoveryInfoType | None = None,
-) -> None:
+) -> bool:
```

Comment:
If the `icloud` method doesn't do anything, should we simply drop it?

Reply:
We can do that if it has been deprecated for at least two releases and a warning logged somewhere.

Reply:
As code owners, maybe @Quentame / @nzapponi would consider looking at that in a follow-up PR?

Reply:
For reference, I managed to find the deprecation PR: #28968

Reply:
Ok, then we can drop this, I think, since the device tracker integration have logged an error for this platform since then.

Reply:
I noticed that the main setup wasn't logging a warning. I have created the corresponding PR #63875 and I think wait until 2022.4 to remove this.